README.md

```diff
@@ -571,12 +582,52 @@ config/setup functions to bytecode, which has this limitation.
 Additionally, if functions are given for these keys, the functions will be passed the plugin
 name and information table as arguments.
 
+### Lockfile
+
+`packer` provides a `lockfile` to help manage plugin updates. This is useful for users that store their
+configuration in some sort of source repository. Committing packer's lockfile will ensure that packer will
+`install` and `update` plugins to known working commits for their configuration.
+
+Enabling lockfile support will change the default behavior of `install`, `update`, and `sync` commands.
+If the lockfile contains a plugin, packer will update to that commit instead of the latest changes. If the
+plugin is not found in the lockfile packer will fetch the latest changes.
+
+When wanting to update your local plugins to the latest changes call your packer update command with
+`--nolockfile` argument. This will ignore the lockfile and update your plugins to the latest changes.
```

Comment:
```suggestion
`packer` can provide a `lockfile` to help manage plugin updates. This is useful for users that store their
configuration in some sort of source repository. Committing the lockfile will ensure that `packer` will
`install` and `update` plugins to known working commits for their configuration.

Enabling lockfile support will change the default behavior of `packer.install()`, `packer.update()`, and
`packer.sync()`. If the lockfile contains a plugin, `packer` will update to the specified commit instead
of the latest changes. If the plugin is not found in the lockfile, `packer` will fetch the latest changes.

If you want to update your local plugins to the latest changes, call `:PackerUpdate` or `:PackerSync` with
the `--nolockfile` argument. This will ignore the lockfile and update your plugins to the latest changes.
```
not sure whether to write as `packer` or Packer